assets/sass/_planetPortrait.scss

```diff
@@ -18,4 +18,12 @@
       background-color: pink;
     }
   }
-}
\ No newline at end of file
+
+  .img--hidded {
+    display: none;
+  }
+
+  .btn-section--active {
+    background-color: cyan;
+  }
```

Comment:
In this class please you can add styles to a selected section button 

<img width="439" alt="image" src="https://user-images.githubusercontent.com/26731448/180100864-bf1ba205-20f4-42b4-ba0b-42b45e7e85a1.png">